app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  belongs_to :post
+
+  def increment_likes_counter
+    post.increment!(:likes_counter)
+  end
```

Comment:
- [ ] [OPTIONAL] You can use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the `#increment_likes_counter` method every time a new post has been saved to the database, then make the `#increment_likes_counter` method private